homeassistant/components/ezviz/camera.py

```diff
@@ -32,166 +50,194 @@
     {
         vol.Required(CONF_USERNAME): cv.string,
         vol.Required(CONF_PASSWORD): cv.string,
-        vol.Optional(CONF_CAMERAS, default={}): {cv.string: CAMERA_SCHEMA},
+        vol.Optional(CONF_REGION, default=DEFAULT_REGION): cv.string,
+        vol.Optional(ATTR_CAMERAS, default={}): {cv.string: CAMERA_SCHEMA},
     }
 )
 
+_LOGGER = logging.getLogger(__name__)
+
+MIN_TIME_BETWEEN_SESSION_RENEW = timedelta(seconds=90)
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
-    """Set up the Ezviz IP Cameras."""
 
-    conf_cameras = config[CONF_CAMERAS]
+async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
+    """Set up a Ezviz IP Camera from platform config."""
+    _LOGGER.warning(
+        "Loading ezviz via platform config is deprecated, it will be automatically imported. Please remove it afterwards."
+    )
 
-    account = config[CONF_USERNAME]
-    password = config[CONF_PASSWORD]
+    if ATTR_CAMERAS in config:
+        cameras_conf = config.get(ATTR_CAMERAS, CAMERA_SCHEMA)
 
-    try:
-        ezviz_client = EzvizClient(account, password)
-        ezviz_client.login()
-        cameras = ezviz_client.load_cameras()
+        _LOGGER.debug("camera constant: %s", cameras_conf)
```

Comment:
Don't log the config. It contains credentials.

Reply:
Removed debug strings.